### PR TITLE
Commerce 12719 1

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://liferay.atlassian.net/browse/COMMERCE-12719

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-client-extension-web/src/main/resources/META-INF/resources/checkout_step/cx_checkout_step.jsp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+String renderURL = (String)request.getAttribute("renderURL");
+%>
+
+<div class="form-group form-group-item input-String-wrapper has-error" id="customCheckoutStepContainer"></div>
```
Comment (alessiorendina) [inline modules/apps/commerce/commerce-client-extension-web/src/main/resources/META-INF/resources/checkout_step/cx_checkout_step.jsp:14]: Try it without this 2 classes `input-String-wrapper has-error`

**Review by alessiorendina — CHANGES_REQUESTED** [review]
@andrea-ale-sbarra Just 1 small fix, thank you!

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:relevant

**Comment by andrea-ale-sbarra** [comment]
ci:forward:force

**Pull request closed without merging**